src/components/Footer.jsx

```diff
@@ -1,155 +1,115 @@
+
+
```

Comment:
remove this whole footer.jsx file change. I believe this is not your changes.

Else look good. Will merge after this.

Reply:
@Mustafiz04 I have removed the footer.jsx file . If there is any more problems let me know sir, I will change that